app/releases/page.tsx

```diff
@@ -26,43 +31,52 @@ export default async function Page() {
             key={release.createdAt}
             className="flex flex-col rounded-lg border shadow-sm dark:border-secondary-700"
           >
-            <div className="flex items-center justify-between p-6 pb-0 pt-4">
+            <div className="flex items-center justify-between p-3 pb-0 pt-4 sm:p-6">
               <div className="flex items-center">
-                <a
-                  href={`https://github.com/coronasafe/${release.repository}`}
-                  target="_blank"
-                  className={`font-mono font-bold tracking-wide text-secondary-700 dark:text-secondary-300`}
+                <h3
+                  className={` justify-center  text-lg font-semibold sm:flex sm:text-2xl`}
                 >
-                  <span className="pr-0.5 tracking-normal text-secondary-400">
-                    {env.NEXT_PUBLIC_GITHUB_ORG}/{release.repository}
-                  </span>
-                </a>
+                  {release.repository}
+                  {" - "}
+                  <GoTag
+                    className="relative hidden pt-1 text-lg sm:inline sm:pt-1.5 "
+                    size={30}
+                    color="green"
+                  />
+                  {release.name}
+                  {myyRepos.has(release.repository) &&
+                    myyRepos.delete(release.repository) && (
+                      <span className="relative ml-2 rounded-full border border-green-700 p-1 pt-2.5 text-xs text-green-700 ">
+                        Latest
+                      </span>
```

Comment:
the "latest" chip has improper padding. this does not look nice...

<img width="415" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/c1c91bec-31ac-4a8e-baad-3184815710bb">


```suggestion
                      <span className="relative ml-2 rounded-full border border-green-700 p-1 text-xs text-green-700 ">
                        Latest
                      </span>
```